www/docs/schemas/adapters.md

```diff
@@ -7,6 +7,24 @@ An **Adapter** in NextAuth.js connects your application to whatever database or
 
 You do not need to specify an Adapter explicitly unless you want to use advanced options such as custom models or schemas, if you want to use the Prisma Adapter instead of the default TypeORM Adapter, or if you are creating a custom Adapter to connect to a database that is not one of the supported databases.
 
+We have recently begun moving the adapter code out of the core of `nextauthjs/next-auth` and into their own repository under [`nextauthjs/adapters`](https://github.com/nextauthjs/adapters).
+
+There you can find the following adapters in various stages of maturity:
+
+- TypeORM
+- TypeORM-Legacy
+- Prisma
+- Prisma-Legacy
+- FaunaDB
+- DynamoDB
+- PouchDB (Active PR)
+- Sanity (Active PR)
+- Sequelize (Active PR)
+
```

Comment:
Auto-generated this list somehow? Have a single source of truth for adapters somewhere.

Reply:
Yeah probably also want to autogenerate this eventnually. I addressed all your comments and actually removed the ones I had listed here with (Active PR) because that stuff can change so quickly.. I just kept the actually active ones on that list for now - which made it shorter and easier to deal with manually for now anyway :+1: